docs/migration/uninstalling_legacy.md

```diff
@@ -24,7 +24,7 @@ If you've used Homebrew to install espanso, you can open a terminal and run the
 
 ```bash
 espanso stop
-espanso unregister
+espanso service unregister
```

Comment:
Thank you. I guess the same applies to the second snippet at https://espanso.org/docs/migration/uninstalling_legacy/#uninstalling-on-macos, and also the item in https://espanso.org/docs/migration/uninstalling_legacy/#uninstalling-on-linux.

I haven't spent much time on that page!

There's another at https://espanso.org/docs/legacy/install/linux/#installing-on-ubuntu--debian.

Would you like to amend those too? If not, I can do them all.

Reply:
On second thought, these instructions relate to uninstalling Espanso Legacy. It's possible the CLI commands were different. I haven't done any investigating of that but there's a line in https://espanso.org/docs/legacy/install/linux/#installing-on-ubuntu--debian which refers to `espanso unregister` so the syntax may be correct for that version?

Reply:
I know that the cli has some "aliases" between sub commands. For example of you `espanso install` means You want to `espanso package install` and so on.
I'm on the phone, I'll back when the job isn't very heavy

Reply:
for the old version 0.7.3, `unregister` is meant for systemd [source](https://github.com/espanso/espanso/blob/386a351df7af1ad3afa50de088a80f4ffefebd19/src/main.rs#L151)

And for the latest version, I was remembering wrong, [the aliases I have read](https://github.com/espanso/espanso/blob/dev/espanso/src/main.rs#L77) don't affect `register` in any way.
Browsing at the code I found two places for `unregister`: `espanso env-path unregister` (the PATH) and `espanso service unregister`(the service).

so the correct way is `espanso unregister` in 0.7.3